.github/workflows/arbitrator-ci.yml

```diff
@@ -120,6 +121,8 @@ jobs:
 
       - name: Install Foundry
         uses: foundry-rs/foundry-toolchain@v1
+        with:
+          cache: false
```

Comment:
why turn this cache off?

Reply:
this is what was done upstream - I'd imagine disabling it was a bandaid fix than an actual fix!